src/lib/ui/core/Dialog/vaul.ts

```diff
@@ -1,4 +1,4 @@
 import { getAllContexts } from 'svelte'
 
-export const getVaulCtx = (ctx = getAllContexts()): [] | [Symbol, any] =>
-  Array.from(ctx.entries()).find(([key, value]) => value?.states?.drawerId) || []
+export const getVaulCtx = (ctx = getAllContexts()): [] | [symbol, any] =>
+  Array.from(ctx.entries()).find(([_key, value]) => value?.states?.drawerId) || []
```

Comment:
It was changed because eslint error. `Symbol` letter case changed automatically by `eslint --fix`